### PR TITLE
Fix/PLF-8156: Fix IE Encoding Issue For Image Select

### PR DESCRIPTION
**Pull request by hbenali**
For IE 11 there is an encoding issue in the CKEditor Image Selector

### CODE REVIEW

File: commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js [hunk 1]
```diff
@@ -580,7 +580,7 @@
                                                                        } else {
                                                                             nodePath = decodeURIComponent(addressBarContent);
                                                                        }
-                                                                       window.open('/eXoWCMResources/eXoPlugins/content/content.html?insertContentType=Image&subPath='+subPath+'&nodePath=' + nodePath + '&viewType=thumbnail&components=' + previewImageId + '&currentInstance=' + editor.name, 'WCMGadgetSelector', 'width=1024,height=600');
+                                                                       window.open(encodeURIComponent('/eXoWCMResources/eXoPlugins/content/content.html?insertContentType=Image&subPath='+subPath+'&nodePath=' + nodePath + '&viewType=thumbnail&components=' + previewImageId + '&currentInstance=' + editor.name, 'WCMGadgetSelector', 'width=1024,height=600'));
```
Comment (thomasdelhomenie) [inline commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: Shouldn't we encode only subPath and nodePath ?

Comment (hbenali) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: The issue is located only on the subpath ... more specific the "space" character.

Comment (thomasdelhomenie) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: Can't we have a node path containing special characters ?

Comment (hbenali) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: Yes, for IE, it does not encode the space character. In the Jira description, there is a problem with "é" character (from the folder name). In fact, it is not the source problem. the space character does. By encoding the URL and setting IE to encode the request URL into UTF-8. This problem is gone. Also, there is no side-effects on the other browsers (chrome, firefox,...)

Comment (thomasdelhomenie) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: ok but I don't get why we need to encode the whole URL, there is no special character in it, except possibly in the subpath and the nodePath.

Comment (hbenali) [reply commons-extension-webapp/src/main/webapp/ckeditor/plugins/image/dialogs/image.js]: changes applied

**Pull request merged**